lib/gollum/templates/wiki_content.mustache

```diff
@@ -0,0 +1,52 @@
+<div id="wiki-content">
+	<h1 class="pt-4">{{page_header}}</h1>
+	<div class="breadcrumb">{{{breadcrumb}}}</div>
+	
+
+	<div class="{{#has_header}}has-header{{/has_header}}{{#has_footer}} has-footer{{/has_footer}}{{#has_sidebar}} has-sidebar has-{{bar_side}}bar{{/has_sidebar}}{{#has_toc}} has-toc{{/has_toc}}">
+	  {{#has_toc}}
+	  <div id="wiki-toc-main">
+	    {{{toc_content}}}
+	  </div>
+	  {{/has_toc}}
+	  {{#has_sidebar}}
+	  <div id="wiki-sidebar" class="gollum-{{sidebar_format}}-content">
+	    <div id="sidebar-content" class="Box Box--condensed col-3 markdown-body pl-4 float-right">
+	      {{{sidebar_content}}}
+	    </div>
+	  </div>
+	  {{/has_sidebar}}
+	  <div id="wiki-body" class="gollum-{{format}}-content">
+	    {{#has_header}}
+	    <div id="wiki-header" class="gollum-{{header_format}}-content">
+	      <div id="header-content" class="markdown-body">
+	        {{{header_content}}}
+	      </div>
+	    </div>
+	    {{/has_header}}
+	    <div class="markdown-body {{#header_enum?}}header-enum{{/header_enum?}}" {{#header_enum?}}style="--header-enum-style:{{header_enum_style}};"{{/header_enum?}}>
```

Comment:
Inline CSS

Reply:
This was also already in place. Not sure how to deal with it in a better way quickly, so I think we should leave this for now.